controller/system_backup_controller.go

```diff
@@ -903,7 +902,7 @@ func (c *SystemBackupController) generateSystemBackupYAMLsForKubernetes(dir stri
 		return
 	}
 
-	err = c.generateSystemBackupYAMLsForPodSecurityPolicy(dir, "roles", "rolebindings", "podsecuritypolicies", scheme)
+	err = c.generateSystemBackupYAMLsForRoles(dir, "roles", "rolebindings", scheme)
```

Comment:
As far as I can tell, there's no reason to back up Roles or RoleBindings except for PodSecurityPolicies. I think we can just remove the additional Role and RoleBinding logic here and elsewhere. I'll defer to you and @c3y1huang if this is incorrect.

I suggest we follow up in a new issue so we can merge this one, however.

Reply:
We also mention that we backup PodSecurityPolicies in the online documentation. We should fix this as part of a followup issue.

Reply:
As a quick sync with @c3y1huang, we might need the `RoleBinding` for the support-bundle.

>I suggest we follow up in a new issue so we can merge this one, however.

I agree.
We could follow up in a new issue.

>We also mention that we backup PodSecurityPolicies in the online documentation. We should fix this as part of a followup issue.

I will update the documents with this ticket https://github.com/longhorn/longhorn/issues/8056